torch/_functorch/vmap.py

```diff
@@ -274,10 +274,16 @@ def vmap_impl(func, in_dims, out_dims, randomness, chunk_size, *args, **kwargs):
         return _chunked_vmap(func, flat_in_dims, chunks_flat_args,
                              args_spec, out_dims, randomness, **kwargs)
 
-    # If chunk_size is not specified.
-    return _flat_vmap(
-        func, batch_size, flat_in_dims, flat_args, args_spec, out_dims, randomness, **kwargs
-    )
+    from torch._dynamo import disable
+
+    @disable
```

Comment:
Remember to remove the disable() when you get torch.compile to work with torch.vmap